.gitignore

```diff
@@ -0,0 +1,76 @@
+# Node modules get installed here
```

Comment:
What is the project this template is based on?

Reply:
I used the gitignore from [ATAT](https://github.com/dod-ccpo/atst) as a starting point, some of it might not apply to us depending on what we go with but I don't think it hurts to have extra.